rfc/text/advanced/ap_10_rpc_progressive_call_invocations.md

```diff
@@ -1,25 +1,25 @@
-## Progressive Calls {#rpc-progressive-calls}
+## Progressive Call Invocations {#rpc-progressive-call-invocations}
 
-A *Caller* may issue a progressive call. This can be useful in a few cases:
+A *Caller* may issue a progressive call with progressive call invocations. This can be useful in a few cases:
 
 * Payload is too big to send it whole in one request, e.g., uploading a file.
 * Long-term data transfer that needs to be consumed early, such as a media stream.
 * RPC is called too often and overall processing can be optimized: avoid the need to generate a new id for requests,
   initiate data structures for a new call, etc.
 
-In such cases, a procedure implemented by a *Callee* and registered at a *Dealer* may be made to receive progressive calls,
-where the *Callee* waits after receiving the entire set of payload chunks before sending the result. 
+In such cases, a procedure implemented by a *Callee* and registered at a *Dealer* may be made to receive progressive call invocations,
+where the *Callee* waits after receiving the entire set of payload chunks before sending the result.
```

Comment:
```suggestion
where the *Callee* may start processing the incoming data without awaiting the entire set of payload chunks.
```

Reply:
@oberstet @ecorm What do you think of this change?

Reply:
it seems the above 2 paragraphs are to describe the 2 options a callee that receives progressive call invocations (not "payload chunks") does have. start processing _and_ start producing (progressive) results immediately OR start receving progressive invocations and processing (or not) immediately, but continue to do so before finally producing 1 non-progressive results.

if so, I agree: it makes sense to explicitly describe those differing behaviors possible for a callee

Reply:
agree! 👍🏻

Reply:
> What do you think of this change?

LGTM